src/plugin-manager.js

```diff
@@ -8,7 +8,7 @@ import {FrameworkConfiguration} from 'aurelia-framework';
  * @export
  * @interface PluginDefinition
  */
-export interface PluginDefinition {
+interface PluginDefinition {
```

Comment:
I'm really trying to finish my projects so I can give this a go. Sorry for the delay (on both PRs).